src/SprykerSdk/Zed/AppSdk/Business/AsyncApi/Builder/AsyncApiBuilder.php

```diff
@@ -390,4 +389,28 @@ protected function orderAsyncApiElements(array $asyncApi): array
 
         return $asyncApi;
     }
+
+    /**
+     * @param string $targetFile
+     * @param array $asyncApi
+     *
+     * @return void
+     */
+    private function updateFile(string $targetFile, array $asyncApi): void
+    {
+        if(file_exists($targetFile)) {
+            $getContent = Yaml::parse(file_get_contents($targetFile));
+            $getContent['info']['version'] = $asyncApi['info']['version'];                
+        }
+                
+        $asyncApiSchemaYaml = Yaml::dump($getContent, 100);
+
+        $dirname = dirname($targetFile);
+
+        if (!is_dir($dirname)) {
+            mkdir($dirname, 0777, true);
+        }
+
+        file_put_contents($targetFile, $asyncApiSchemaYaml);
```

Comment:
You should re-use the existing `\SprykerSdk\Zed\AppSdk\Business\AsyncApi\Builder\AsyncApiBuilder::writeToFile()` method.